### ISSUE TITLE
ssl port alternative.

### ISSUE THREAD

**Issue by burghy86** [janeczku/calibre-web#2050]
**Describe the bug/problem**
good morning. if I create a docker image with Technosoft2000 to reach it remotely I created a reverse dns (example public port ssl https 1234 to internal docker port http 8083) everything works.
if I set the reverse proxy on caliber-web setting and as public port 1234 to set the kobo sync the generated kobo token uses the correct port but generates everything in http and therefore gives an error.
if I change it to https and insert it in the kobo config he still generates the url with http and the sync does not work.
could you put a flag to set ssl or not ssl to the kobo token?
sorry fo my bad english
p.s. 
I use reverse dns for https security and lets encrypt certificate generated by synology

**Comment by OzzieIsaacs** [comment]
For unknown reason may people having problems setting up ngnix proxy. 
Please check our wiki. Please make sure you have added the  X-Scheme line to your proxy config, this makes calibre-web aware of ssl connections.
`proxy_set_header        X-Scheme        $scheme;`

Please also make sure to check the additional hint on non standard ssl ports for nginx config.
https://github.com/janeczku/calibre-web/wiki/Setup-Reverse-Proxy#nginx

If this doesn't solve your problem, please try to find an answer on the internet yourself. I'm tired of setting up an ngnix server coping the config from the wiki and realising time after time that the example is just working without any problem. 
I don't know your config, but you know my config (from the example)!

**Comment by burghy86** [comment]
Hello. sorry if i tired you with yet another proxy question.
i solved with synology. I kept looking the wrong way

I added the two strings of the wiki and now everything works in https too.

thanks again for your patience 
![proxy2](https://user-images.githubusercontent.com/17832100/125925967-afe8df32-3fac-4b0e-a7b9-6c6beb666eab.PNG)
![reverse1](https://user-images.githubusercontent.com/17832100/125925978-43ab7cb1-0ca1-466f-92c1-8031d7445ec1.PNG)
